src/Rules/Properties/AccessPropertiesCheck.php

```diff
@@ -160,6 +160,16 @@ private function processSingleProperty(Scope $scope, PropertyFetch $node, string
 		}
 
 		$propertyReflection = $type->getProperty($name, $scope);
+		if ($propertyReflection->isStatic()) {
+			return [
+				RuleErrorBuilder::message(sprintf(
+					'Non static access to static property %s::$%s.',
```

Comment:
```suggestion
					'Non-static access to static property %s::$%s.',
```

Reply:
For refrence: I stuck with `non static` over `non-static` to match the PHP runtime message, but I do agree this variant is more readable.